### PR TITLE
miscellaneous fixes

### PR DESCRIPTION
**Pull request by perigoso**
- trimmed trailing whitespaces
- fixed end of files (end in one, and only one, ~CR~ newline)
- fixed some spelling errors

### CODE REVIEW

**Comment by dragonmux** [comment]
A quick review says this looks good. Before we approve it for merge, just to check when you said "CR" above, do you actually mean \r, or do you instead mean \n (LF)? The existing files should be mostly using Linux line endings (LF), so lets be consistent and use the same line endings everywhere

**Comment by perigoso** [comment]
Sorry, I mean all files now end in a single newline, some where missing, others had duplicates , line endings were not modified

**Review by dragonmux — APPROVED** [review]
Excellent, LGTM! Merging

**Pull request merged**
